x-pack/plugins/ml/public/application/explorer/explorer_charts/explorer_chart_single_metric.js

```diff
@@ -375,9 +385,17 @@ export class ExplorerChartSingleMetric extends React.Component {
         .enter()
         .append('circle')
         .attr('r', LINE_CHART_ANOMALY_RADIUS)
+        .on('click', function (d) {
+          d3.event.preventDefault();
+          if (d.anomalyScore === undefined) return;
+          showAnomalyPopover(d, this);
```

Comment:
Can you enable the popover on multi-bucket markers too? This is already supported in the Single Metric Viewer chart.

Reply:
Fixed in 7829012.